src/components/Dashboard/UrlListItem.tsx

```diff
@@ -15,48 +16,52 @@ interface UrlListItemProps {
-                <div className="flex shrink items-center  w-[60%]">
-                    <div className="ml-2 sm:ml-4 w-[100%]">
-                        <div className="flex max-w-fit items-center space-x-2 ">
-                            <Link
-                                href={`${TINY_SITE}/${url.ShortUrl}`}
-                                target="_blank"
-                                rel="noreferrer"
-                                className="w-full max-w-[140px] truncate text-sm font-semibold text-blue-800 sm:max-w-[300px] sm:text-base md:max-w-[360px] xl:max-w-[500px]"
-                            >
-                                {TINY_SITE}/{url.ShortUrl}
-                            </Link>
-                            <Button
-                                className="group rounded-full bg-gray-100 p-1.5 transition-all duration-75 hover:scale-105 hover:bg-blue-100 active:scale-95"
-                                onClick={() => copyButtonHandler(url.OriginalUrl)}
-                                testId="copy-button"
-                            >
-                                <span className="sr-only">Copy</span>
-                                <CopyIcon width={15} height={15} />
-                            </Button>
-                        </div>
-                        <div className="flex max-w-fit items-center space-x-1">
-                            <Link
-                                href={url.OriginalUrl}
-                                target="_blank"
-                                rel="noopener noreferrer"
-                                className="xs:block max-w-[200px] truncate text-sm font-medium text-gray-700 underline-offset-2 hover:underline sm:max-w-[300px] md:max-w-[400px] xl:max-w-[440px]"
-                            >
-                                {url.OriginalUrl}
-                            </Link>
-                        </div>
-                    </div>
+            <div className="flex flex-col w-[85%]">
+                <div className="flex max-w-fit items-center space-x-2 ">
+                    <Link
+                        href={`${TINY_SITE}/${url.ShortUrl}`}
+                        target="_blank"
+                        rel="noreferrer"
+                        className="w-full  truncate text-sm font-semibold text-blue-800 sm:max-w-[300px] sm:text-base md:max-w-[360px] xl:max-w-[500px]"
+                    >
+                        {TINY_SITE}/{url.ShortUrl}
+                    </Link>
+                    <Button
+                        className="rounded-full bg-gray-100 p-1.5 transition-all duration-75 hover:scale-105 hover:bg-blue-100 active:scale-95"
+                        onClick={() => copyButtonHandler(url.OriginalUrl)}
+                        testId="copy-button"
+                    >
+                        <span className="sr-only">Copy</span>
+                        <MdOutlineContentCopy style={{ color: 'black' }} />
```

Comment:
can we use className in place of inline css

Reply:
Done